client/app/bundles/course/learning-map/components/Gate/index.jsx

```diff
@@ -0,0 +1,184 @@
+import { connect } from 'react-redux';
+import { selectGate } from 'course/learning-map/actions';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import ConnectionPoint from '../ConnectionPoint';
+import { elementTypes, satisfiabilityTypes } from '../../constants';
+import translations from '../../translations.intl';
+import { nodeShape, selectedElementShape } from '../../propTypes';
+
+const red = '#f08080';
```

Comment:
What are your thoughts on using Material-UI colors instead?

Reply:
Updated to use MUI colors. As for the Hound issue with the `ConnectionPoint` scss, it doesn't work without the `px` values so I left them in, but I moved the color string literal into a variable.